docs/website/packages/codegen/copy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+title: Copying
+description: codegen's copy function
+---
+
+The `copy` function is designed to facilitate the duplication of source code from one FHIR version to another.
+
+## Basic Example
+
+### Set Up Your Source and Target Directories
+
+Ensure that you have two directories: one for the source FHIR version and another for the target FHIR version. For example, you might have a directory structure like this:
+
+- `src/r5` (source)
+- `src/r4b` (target)
+
+### Prepare Conditional Preprocessing
+
+If you need conditional preprocessing (such as including code only for specific FHIR versions), add preprocessing directives to your source files. For example:
+
+```ts
+// #if fhir >= r5
+// Specific code for FHIR version r5 or higher
+```
```

Comment:
This is missing the closing tag (I though I added this comment the first time but I can't find it, so I probably forgot to save - sorry about that).


```suggestion

```ts
// #if fhir >= r5
// Specific code for FHIR version r5 or higher
// #endif
```
```